pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -61,18 +61,32 @@ func launchTemplateName(clusterName string, arch string) string {
 	return fmt.Sprintf(launchTemplateNameFormat, clusterName, arch)
 }
 
-func (p *LaunchTemplateProvider) Get(ctx context.Context, cluster *v1alpha1.ClusterSpec, constraints Constraints) (*ec2.LaunchTemplate, error) {
+func (p *LaunchTemplateProvider) Get(ctx context.Context, cluster *v1alpha1.ClusterSpec, constraints Constraints) (*LaunchTemplate, error) {
+	// If the customer specified a launch template then just use it
+	if desc := constraints.GetLaunchTemplate(); desc != nil {
```

Comment:
what is desc? Description?  Can we just call this launchTemplate or result ?

Reply:
will fix